### PR TITLE
Enh/byol

### PR DESCRIPTION
**Pull request by NiranjanChaudhari0929**
Editing BYOL file

### CODE REVIEW

**Comment by NiranjanChaudhari0929** [comment]
Proceeding with changes in prep-viz command

File: src/lmcontrol/nn/clf.py
Comment (ajtritt) [inline src/lmcontrol/nn/clf.py:1]: Please delete all the lines that are commented out.
Please change "conditions" to "condition"

File: src/lmcontrol/nn/resnet.py [hunk 1]
```diff
@@ -150,10 +151,10 @@ def _forward_impl(self, x: Tensor) -> Tensor:
 
         x = self.avgpool(x)
         x = torch.flatten(x, 1)
-
+
```
Comment (ajtritt) [inline src/lmcontrol/nn/resnet.py:154]:
```suggestion

```

File: src/lmcontrol/nn/resnet.py [hunk 2]
```diff
@@ -150,10 +151,10 @@ def _forward_impl(self, x: Tensor) -> Tensor:
 
         x = self.avgpool(x)
         x = torch.flatten(x, 1)
-
+        
         if not self.return_embeddings:
             x = self.fc(x)
-
+
```
Comment (ajtritt) [inline src/lmcontrol/nn/resnet.py:157]:
```suggestion

```

File: src/lmcontrol/segment.py [hunk 3]
```diff
@@ -285,17 +331,18 @@ def crop_size(string):
             seg_images.append(segi)
             seg_masks.append(segm)
         except UnsegmentableError:
-            if args.save_unseg:
```
Comment (ajtritt) [inline src/lmcontrol/segment.py:288]: Please keep keep save_unseg and make center_crop optional

**Review by ajtritt — CHANGES_REQUESTED** [review]

File: src/lmcontrol/nn/clf.py
Comment (NiranjanChaudhari0929) [reply src/lmcontrol/nn/clf.py:1]: I have removed the commented lines.
I have changed  "conditions" to "condition"

File: src/lmcontrol/segment.py (hunk 3 above)
Comment (NiranjanChaudhari0929) [reply src/lmcontrol/segment.py:288]: done

**Pull request merged**